### PR TITLE
route comments complete

### PR DESCRIPTION
**Pull request by KevinHuang0126**
### Trello Link
[Ticket](https://trello.com/c/dIyEbm9R)

# Description

Added comments to auth, business, and user routes 

# issues found

getting favorite and connected business API has a prisma error (lines 103 and 131)

line 141 has a route that does the same thing as the route on line 38; also the one on line 141 has an API error

### CODE REVIEW

File: src/routes/auth.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import TokenService from '../services/tokenService.js';
 import Logger from '../tools/logger.js';
 
 const router = express.Router();
-
+// signing up for an account
 router.post('/signup', async (request, response, next) => {
```
Comment (Laurabtt152) [inline src/routes/auth.js:14]: Great job on this first part, however, do you understand the (request, response,next)? What is its function?

File: src/routes/business.js [hunk 2]
```diff
@@ -99,7 +99,7 @@ router.delete('/:id', async (request, response, next) => {
     next();
   }
 });
-
+//find user's favorite businesses. this isn't working for me, i got a prisma error
 router.get('/favorites', async (request, response, next) => {
```
Comment (Laurabtt152) [inline src/routes/business.js:103]: Thank you for letting me know, but how did you test this? Did you use Postman? What Prisma error pops up?

File: src/routes/business.js [hunk 3]
```diff
@@ -128,7 +128,7 @@ router.get('/favorites', async (request, response, next) => {
     next(error);
   }
 });
-
+// get the user's connected businesses. this one isn't working for me either, same prisma error
 router.get('/connected', async (request, response, next) => {
```
Comment (Laurabtt152) [inline src/routes/business.js:132]: See comment above.

File: src/routes/user.js [hunk 4]
```diff
@@ -137,7 +137,7 @@ router.delete('/:id', async (request, response, next) => {
     next(error);
   }
 });
-
+// why does this route exist when we have the '/' route already doing the same thing? can we delete this
 router.get('/user', async (request, response, next) => {
```
Comment (Laurabtt152) [inline src/routes/user.js:141]: Which '/' route are you referring to? All routes are specific to their functions, so if another route is doing this first as a check, then we should not delete this.

**Comment by KevinHuang0126** [comment]
Should I just close it instead?

**Pull request closed without merging**